lib/regexp/regexp.go

```diff
@@ -0,0 +1,258 @@
+//
+//     re.find(src) - Returns a string holding the text of the leftmost match in the given string of the regular expression re.
+//                    If there is no match, the return value is an empty string, but it will also be empty if the
+//                    regular expression successfully matches an empty string.
+//
+//     re.find_all(src, max) - Returns a tuple of all successive matches of the regular expression re. An empty tuple indicates
+//                             no match. If max > 0, at most max strings will be returned. If max == 0,
+//                             an empty tuple will be returned. If max < 0, all strings will be returned.
+//                             The parameter max is optional, by default no limit will be applied.
+//
+//     re.find_submatches(src) - Returns a tuple of strings holding the text of the leftmost match of the regular expression re
+//                               in the given string and the matches, if any, of its subexpressions. An empty tuple indicates
+//                               no match.
+//
+//     re.matches(src) - Indicates whether the given string contains any match of the regular expression re.
+//
+//     re.replace_all(src , repl) - Returns a copy of the given string, replacing matches of the regular expression re
+//                                  with the replacement string repl. Inside repl, $ signs are interpreted as in Expand,
+//                                  so for instance $1 represents the text of the first submatch.
+//
+//     re.replace_all(src , replFunc) - Returns a copy of the given string in which all matches of the regular expression re
+//                                      have been replaced by the return value of the replacement function applied to the
+//                                      matched substring. The replacement returned by replacement function is substituted directly,
+//                                      without using Expand.
+//
+//     re.split(src, max) - Returns a tuple of strings between all the matches of the regular expression re.
+//                          If max > 0, at most max strings will be returned knowing that the last string will
+//                          be the unsplit remainder. If max == 0, an empty tuple will be returned. If max < 0,
+//                          all strings will be returned. The parameter max is optional, by default no limit
+//                          will be applied.
+//
+var Module = &starlarkstruct.Module{
+	Name: "regexp",
+	Members: starlark.StringDict{
+		"compile": starlark.NewBuiltin("compile", compile),
+	},
+}
+
+func compile(_ *starlark.Thread, b *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var (
+		pattern string
+	)
+
+	if err := starlark.UnpackPositionalArgs(b.Name(), args, kwargs, 1, &pattern); err != nil {
+		return starlark.None, err
+	}
+
+	if strings.Contains(pattern, `\C`) {
```

Comment:
This is not the correct check, as it will incorrectly reject `\\C` (two backslashes).
It is necessary to build the rudiments of a scanner for RE2 strings to correctly detect this case.